nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -169,6 +174,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
+        session.getProvenanceReporter().send(flowFile, flowFile.getAttribute(HTTP_REQUEST_URI), stopWatch.getElapsed(TimeUnit.MILLISECONDS));
```

Comment:
@markap14 I just updated the PR with the proposed approach (in my previous comment), let me know if it is something more accurate.
Note: I changed URI for both Request and Response since I added a common method usable from both processors. If it sounds OK to you, I'll update the JIRA to reflect the change.